sdk/core/core-rest-pipeline/src/fetchHttpClient.ts

```diff
@@ -0,0 +1,237 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/// <reference lib="dom" />
```

Comment:
[dom is already listed in the tsconfig](https://github.com/Azure/azure-sdk-for-js/blob/b7dab79ca5df25032728b052ba31c5f1c3a39dc8/sdk/core/core-rest-pipeline/tsconfig.json#L9), so is this really needed?